.github/workflows/pr-image.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build and push image from PR
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    name: Build and push image from PR
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Log in to ghcr
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build the image
+        run: make build
+
+      - name: Push PR image to the registry
```

Comment:
Is there anyway to manage the lifecycle of these images so that we don't keep them as long as we keep the released versions?

Reply:
Will check that and follow up of what I find.

Reply:
Used this [action](https://github.com/marketplace/actions/delete-package-versions) to delete all pre-release images.

Reply:
Trying to have a deletion scheme only for the `prbase` images, so far am not confident with the delete-action setting I have in place.